src/settings/index.js

```diff
@@ -9,6 +11,11 @@ import CopyrightStatusSettings from './CopyrightStatusSettings';
 
 export default class CoursesSettings extends React.Component {
   pages = [
+    {
```

Comment:
Why is this called "display settings" when its sole component is the boolean "duplicateCrosslistedCourses" settings which affects behaviour rather than display?

Reply:
Well, I didn't want to name it anything hyper-specific when I feel like other settings could be in here.

I thought about `General settings` at first, which is what I called a similar pane in Agreements a while back. But then I went and looked at that pane now and after a UX review and standardization, it was renamed `Display settings`. So I decided to just match that, since the settings in there are also "behaviour of the display" settings.

Reply:
The thing is, "display settings" is not just generic, it's actively misleading. I think "general" would be better.

_However_ it's probably not worth the hassle of changing code, filenames and translations, so if you want to just shrug and leave it as it is, I'm fine with that.

Reply:
I also think 'General' would be better - I named it that the first time! I was kind of surprised that it was renamed in Agreements. But I'll defer to the UX people's wisdom in this case rather than overriding them without consultation.

Reply:
Yep, that's legit I guess.